### PR TITLE
(SERVER-363) EZBake 0.2.3 support

### PR DESCRIPTION
**Pull request by nwolfe**
This commit should bring the stable branch up to the newest EZBake-as-a-plugin.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -70,6 +81,12 @@
                    ; SERVER-332, enable SSLv3 for unit tests that exercise SSLv3
                    :jvm-opts      ["-Djava.security.properties=./dev-resources/java.security"]}
 
+             :ezbake {:dependencies ^:replace [[puppetlabs/puppet-server ~ps-version]
+                                               [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty-version]
+                                               [org.clojure/tools.nrepl "0.2.3"]]
+                      :plugins [[puppetlabs/lein-ezbake "0.2.3"]]
```
Comment (nwolfe) [inline project.clj]: This is blocked on the 0.2.3 release of ezbake.

**Comment by nwolfe** [comment]
This should be ready for merge now that ezbake 0.2.3 has been released.

**Comment by fpringvaldsen** [comment]
:see_no_evil: Although somebody more familiar with lein-ezbake should probably review this.

**Comment by nwolfe** [comment]
Oh, if you want to try this out locally just do:

1. git clone + checkout the branch
2. `lein with-profile ezbake ezbake build`
3. Wait a couple minutes for it to finish and give you the URL to your package (on builds.puppetlabs.lan)
4. Install the package on a VM as you normally would from builds.puppetlabs.lan

**Comment by fpringvaldsen** [comment]
:+1: I have now functionally reviewed this and ensured that the package I built works on CentOS 7.

**Comment by waynr** [comment]
Where did you pull ezbake/config files from--the `OLD_puppetserver/stable` branch of ezbake?

**Comment by nwolfe** [comment]
> Where did you pull ezbake/config files from--the OLD_puppetserver/stable branch of ezbake?

No, not ezbake, but maybe puppet-server/master and/or puppetdb? If by "pull" you mean copy/paste, I didn't do much of that, but rather tried cobbling together the minimum/required changes from various sources.

**Comment by waynr** [comment]
Just FYI, this is where these files were coming from with the old style of ezbake packaging:
https://github.com/puppetlabs/ezbake/tree/OLD_puppetserver/stable/configs/puppetserver/config

It'd be a good idea to make sure that what goes into puppet-server stable branch now matches this to avoid unintentional behavior changes in the 1.X series of puppetserver.

**Comment by waynr** [comment]
s/matches/is compatible with/

**Comment by nwolfe** [comment]
> It'd be a good idea to make sure that what goes into puppet-server stable branch now matches this to avoid unintentional behavior changes in the 1.X series of puppetserver.

Good call. I just double-checked that things match between the files in this PR and those in ezbake/OLD_puppetserver. The only difference is in the logback.xml file, which is missing a log appender, but there's a separate ticket for that which will be coming in soon (SERVER-356).

**Comment by waynr** [comment]
@nwolfe that ticket looks specific to the `puppetserver foreground` command and seems to be targeted specifically at `puppet-server 2.0.0`, do you think the work for that ticket will be sufficient to get the log appender into logback.xml on this branch?

**Comment by nwolfe** [comment]
@waynr That ticket will be targeted at stable, since the bug is on stable. It will get into master when we/I do the merge from stable to master (or whatever direction we do it in, haven't done one yet :smiley:). Yes, that work will be sufficient - I'm going to re-add the log appender into logback.xml on this branch.

**Comment by waynr** [comment]
:+1:

**Pull request merged**